homeassistant/components/binary_sensor/__init__.py

```diff
@@ -0,0 +1,38 @@
+"""
+homeassistant.components.binary_sensor
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Component to interface with binary sensors (sensors which only know two states)
+that can be monitored.
+
+For more details about this component, please refer to the documentation at
+https://home-assistant.io/components/binary_sensor/
+"""
+import logging
+
+from homeassistant.helpers.entity_component import EntityComponent
+from homeassistant.components import wink, zwave, isy994, verisure
+
+DOMAIN = 'binary_sensor'
+DEPENDENCIES = []
+SCAN_INTERVAL = 30
+
+ENTITY_ID_FORMAT = DOMAIN + '.{}'
+
+# Maps discovered services to their platforms
+DISCOVERY_PLATFORMS = {
+    wink.DISCOVER_SENSORS: 'wink',
```

Comment:
Please remove the discovery platforms as these platforms do not exist inside binary sensor.